joomla/joomla-cms
# Helpfix

Pull Request for Issue # .

### Summary of Changes

This pull request adds links to two new Help screens and fixes a problem with the Help button in the Who's online module

### Testing Instructions

1. Go to the **Modules: Privacy Status Check** page and click the Help button in the Toolbar
2. Go to the **Modules: Sample Data** page and click the Help button in the Toolbar
3. Go to the **Modules: Who's Online** page and click the Help button in the Toolbar

### Actual result BEFORE applying this Pull Request

1. 404 Not Found
2. 404 Not found
3. Help button inactive

### Expected result AFTER applying this Pull Request

All deliver working Help pages

### Documentation Changes Required

None
